src/core/lookup.c

```diff
@@ -216,14 +227,14 @@ QuicLookupRebalance(
                     }
                     CxPlatHashtableRemove(&PreviousTable[i].Table, Entry, NULL);
 
-                    QUIC_CID_HASH_ENTRY *CID =
+                    QUIC_CID_HASH_ENTRY* CID =
                         CXPLAT_CONTAINING_RECORD(
                             Entry,
                             QUIC_CID_HASH_ENTRY,
                             Entry);
                     (void)QuicLookupInsertLocalCid(
                         Lookup,
-                        CxPlatHashSimple(CID->CID.Length, CID->CID.Data),
+                        CxPlatHashSimple(CID->CID->CID.Length, CID->CID->CID.Data),
```

Comment:
`CID->CID->CID` is a bit ridiculous. 😄 We should figure out how to get some more descriptive names.

Reply:
I agree with you. Now, we use HashEntry instead of CID. And we renamed the pointer of `QUIC_CID_SLIST_ENTRY` in `QUIC_CID_HASH_ENTRY` from `CID` to `Parent`.